### PR TITLE
added chest object

### PR DESCRIPTION
**Pull request by codewithsupra**
Added a chest object that gets more points

### CODE REVIEW

File: public/assets/sprites/chest.png
Comment (r4pt0s) [inline public/assets/sprites/chest.png:1]: This is not really a sprite. If looks more like a normal image which doesn't fit into the style which the entire game follows. Can you replace it with a 2D top-down pixel chest sprite?

File: src/gameObjects/map_start/chest.gameObject.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const chest = (k, map, spawnpoints) => {
+    k.loadSprite('chest', './assets/sprites/chest.png');
+  
+    return k.add([
+      k.sprite('chest'),
+      k.pos(map.pos.x + spawnpoints.chest.x, map.pos.y + spawnpoints.chest.y),
```
Comment (r4pt0s) [inline src/gameObjects/map_start/chest.gameObject.js:6]: Unfortunately, there is no spawnpoint.chest available for now.
Have a look at other game object files to get an idea how you can place objects differently in such a case

File: yarn.lock
Comment (r4pt0s) [inline yarn.lock:1]: Please remove the yarn.lock file because we use npm primarily for the project.

**Review by r4pt0s — CHANGES_REQUESTED** [review]
@codewithsupra there are a few changes to make in your PR.
Also, make sure you check the game with your changes locally with
```bash
npm run dev
```
To make sure there are no errors.

File: yarn.lock
Comment (codewithsupra) [reply yarn.lock:1]: i have added as a new branch. please check

**Pull request closed without merging**